etc/checkfmt.sh

```diff
@@ -1,12 +1,15 @@
 #!/bin/bash
 
+go install golang.org/x/tools/cmd/goimports
+
 HAS_FMT_ERR=0
 # For every Go file in the project, excluding vendor...
-for file in $(go list -f '{{$dir := .Dir}}{{range .GoFiles}}{{printf "%s/%s\n" $dir .}}{{end}}' ./...); do
+
+for file in $(go list -f '{{$dir := .Dir}}{{range .GoFiles}}{{printf "%s/%s\n" $dir .}}{{end}}{{range .TestGoFiles}}{{printf "%s/%s\n" $dir .}}{{end}}{{range .IgnoredGoFiles}}{{printf "%s/%s\n" $dir .}}{{end}}{{range .CgoFiles}}{{printf "%s/%s\n" $dir .}}{{end}}' ./... ); do
```

Comment:
Even more fun - we were not doing format checking on tests, cgo files, or ignored files like `tools.go`